apps/doji-frontend/src/pages/schedule-session/index.tsx

```diff
@@ -0,0 +1,268 @@
+                /hr/person
+              </Typography>
+            </Grid>
+            <Grid item xs={12} marginBottom={3}>
+              <Typography variant="regular" fontWeight={400}>
+                {serviceData.description}
+              </Typography>
+            </Grid>
+            <Grid item xs={12} marginBottom={3}>
+              <LocalizationProvider dateAdapter={AdapterDateFns}>
+                <Controller
+                  name="date"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <DatePicker
+                      {...register('date')}
+                      disablePast
+                      label="Date"
+                      value={value}
+                      onChange={onChange}
+                      renderInput={(params) => (
+                        <TextField
+                          {...params}
+                          error={!!errors.date}
+                          helperText={errors.date?.message}
+                          fullWidth
+                        />
+                      )}
+                    />
+                  )}
+                ></Controller>
+              </LocalizationProvider>
+            </Grid>
+            <Grid item xs={6} marginBottom={3}>
+              <LocalizationProvider dateAdapter={AdapterDateFns}>
+                <Controller
+                  name="startTime"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <TimePicker
+                      {...register('startTime')}
+                      label="Start Time"
+                      value={value}
+                      onChange={onChange}
+                      renderInput={(params) => (
+                        <TextField
+                          {...params}
+                          error={!!errors.startTime}
+                          helperText={errors.startTime?.message}
+                        />
+                      )}
+                    />
+                  )}
+                ></Controller>
+              </LocalizationProvider>
+            </Grid>
+            <Grid item xs={6} marginBottom={3}>
+              <LocalizationProvider dateAdapter={AdapterDateFns}>
+                <Controller
+                  name="endTime"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <TimePicker
+                      {...register('endTime')}
+                      label="End Time"
+                      value={value}
+                      onChange={onChange}
+                      renderInput={(params) => (
+                        <TextField
+                          {...params}
+                          error={!!errors.endTime}
+                          helperText={errors.endTime?.message}
+                        />
+                      )}
```

Comment:
These two controller is very similar. can you create a reusable component for this